functionnal_tests/cypress/integration/activity_feed/hidden_attached_file_activity_spec.js

```diff
@@ -0,0 +1,35 @@
+import { PAGES } from '../../support/urls_commands'
+
+let workspaceId
+let publicationId
+const publicationLabel = 'publication'
+const fileTitle = 'FileTitle'
+const fullFilename = 'Linux-Free-PNG.png'
+const contentType = 'image/png'
+
+describe('Recent activities page', () => {
+  before(function () {
+    cy.resetDB()
+    cy.setupBaseDB()
+    cy.loginAs('administrators')
+    cy.fixture('baseWorkspace').as('workspace').then(workspace => {
+      workspaceId = workspace.workspace_id
+      cy.createPublication(publicationLabel, workspaceId).then(publication => {
```

Comment:
I see "createPublication", I'm then confused by which name it actually is in the code